pom.xml

```diff
@@ -2317,16 +2300,16 @@
 
   <distributionManagement>
     <repository>
+      <id>nexus</id>
+      <name>OSGeo Release Repository</name>
+      <url>https://repo.osgeo.org/repository/geotools-releases/</url>
       <uniqueVersion>false</uniqueVersion>
-      <id>osgeo</id>
-      <name>Open Source Geospatial Foundation - WebDAV upload</name>
-      <url>dav:http://download.osgeo.org/upload/geotools/</url>
     </repository>
     <snapshotRepository>
-        <id>boundless</id>
-        <uniqueVersion>false</uniqueVersion>
-        <name>Boundless Snapshot Repository</name>
-        <url>https://repo.boundlessgeo.com/snapshot/</url>
+      <id>nexus</id>
```

Comment:
Does this id name need to be different from the release repo above? geotools-release and geotools-snapshot for example?

Reply:
@tbarsballe I would like to use consistent naming for these repositories to make the documentation easier to follow.

Can we make the change now?